test.js

```diff
@@ -1,36 +1,59 @@
 'use strict';
-var assert = require('assert');
+var assert = require('chai').assert;
```

Comment:
Some follow-up information, the failing test is making a request to the following URL: `https://registry.npmjs.org/@sindresorhus%2Fdf/1.0.1`

That URL is returning the HTTP status code `401 Unauthorized`. Furthermore, in the response headers is the following item: `npm-notice:ERROR: you cannot fetch versions for scoped packages`

Making similar requests to un-scoped packages work just fine.